feature_layers/feature-collection-layer-from-portal/src/main/java/com/esri/samples/feature_collection_layer_from_portal/FeatureCollectionLayerFromPortalSample.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2016 Esri.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.feature_collection_layer_from_portal;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+import java.io.IOException;
+
+public class FeatureCollectionLayerFromPortalSample extends Application {
+
+  private static FeatureCollectionLayerFromPortalController controller;
+
+  @Override
+  public void start(Stage stage) throws IOException {
+    // set up the scene
+    FXMLLoader loader = new FXMLLoader(getClass().getResource("/fclfp.fxml"));
```

Comment:
I don't think we need to keep the MVC structure here, but just pointing out that it would be best to name the `fxml` file with a more clear file name (generally the samples name with underscores), so `feature_collection_layer_from_portal.fxml`